bootloader/src/nRF52840/usbd_device.cpp

```diff
@@ -146,6 +146,8 @@ class NrfDevice : public Device {
 # define USBD_PRODUCT_NAME              "B SoM"
 #elif PLATFORM_ID == 25
 # define USBD_PRODUCT_NAME              "B5 SoM"
+#elif PLATFORM_ID == 26
+# define USBD_PRODUCT_NAME              "Asset Tracker"
```

Comment:
We need to change "Asset Tracker" to "Tracker" per Ian.

Reply:
@eberseth Can I ask you to confirm what all of these should be:

1. The thing this comment links to which affects the USB string in DFU mode: "%USBD_PRODUCT_NAME DFU Mode"
2. Internal platform product description: https://github.com/particle-iot/device-os/blob/feature/asset-tracker/build/platform-id.mk#L328
3. USB product USB string: https://github.com/particle-iot/device-os/blob/feature/asset-tracker/hal/src/tracker/app_usbd_string_config.h#L85
4. USB Control interface / CDC (serial) USB strings: https://github.com/particle-iot/device-os/blob/feature/asset-tracker/hal/src/tracker/app_usbd_string_config.h#L127
5. How we mention AT in the release list: https://github.com/particle-iot/device-os/blob/feature/asset-tracker/system/system-versions.md

Reply:
1. "Tracker"
2. "Production Tracker"
3. "Tracker CDC Mode"
4. "Tracker Serial"
5. "Tracker"

Reply:
Fixed.